analyze_openapi_changes.py

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+"""Analyze OpenAPI schema changes and generate a structured Markdown report."""
+
+import os
+import json
+import argparse
+from datetime import datetime
+from deepdiff import DeepDiff
+from typing import Dict, Any, Optional
+
+def load_json_file(file_path: str) -> Optional[Dict[str, Any]]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)